examples/python/python3/HistoricalDataRequest.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+# usage: python3 HistoricalDataRequest.py <host-ip>
+
+import argparse
+import json
+import ssl
+import sys
+import urllib.request
```

Comment:
I have tried `import urllib` but got the error `AttributeError: 'module' object has no attribute 'request'`. `import urllib.request` worked well tho.